task/control-plane-agent/src/inventory.rs

```diff
@@ -327,31 +336,30 @@ mod max_num_devices {
         }
         x
     }
-}
 
-// We will spread the contents of `DEVICES` out over multiple packets to MGS;
-// however, we do _not_ currently handle the case where a single `DEVICES` entry
-// is too large to fit in a packet, even if it's the only device present in that
-// packet. Therefore, we assert at compile time via all the machinery below that
-// each entry of `DEVICES` is small enough that it will indeed fit in one packet
-// after being packed into a TLV triple.
-const ASSERT_EACH_DEVICE_FITS_IN_ONE_PACKET: () =
-    assert_each_device_tlv_fits_in_one_packet();
-
-const fn assert_device_tlv_fits_in_one_packet(
-    device: &'static str,
-    description: &'static str,
-) {
-    use gateway_messages::{tlv, SerializedSize, MIN_TRAILING_DATA_LEN};
-
-    let encoded_len = tlv::tlv_len(
-        gateway_messages::DeviceDescriptionHeader::MAX_SIZE
-            + device.len()
-            + description.len(),
-    );
+    // We will spread the contents of `DEVICES` out over multiple packets to
+    // MGS; however, we do _not_ currently handle the case where a single
+    // `DEVICES` entry is too large to fit in a packet, even if it's the only
+    // device present in that packet. Therefore, we assert at compile time via
+    // all the machinery below that each entry of `DEVICES` is small enough that
+    // it will indeed fit in one packet after being packed into a TLV triple.
+    pub(super) const ASSERT_EACH_DEVICE_FITS_IN_ONE_PACKET: () =
+        assert_each_device_tlv_fits_in_one_packet();
+
+    const fn assert_device_tlv_fits_in_one_packet(
+        device: &'static str,
+        description: &'static str,
+    ) {
+        use gateway_messages::{tlv, SerializedSize, MIN_TRAILING_DATA_LEN};
+
+        let encoded_len = tlv::tlv_len(
+            gateway_messages::DeviceDescriptionHeader::MAX_SIZE
+                + device.len()
+                + description.len(),
+        );
 
-    if encoded_len > MIN_TRAILING_DATA_LEN {
-        panic!(concat!(
+        if encoded_len > MIN_TRAILING_DATA_LEN {
+            panic!(concat!(
```

Comment:
Want to also indent the inner text here?

Reply:
Nice catch, fixed in 664ccbcf